shell/platform/windows/platform_view_manager.cc

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/platform_view_manager.h"
+
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/standard_method_codec.h"
+
+namespace flutter {
+
+namespace {
+constexpr char kChannelName[] = "flutter/platform_views";
+}
+
+PlatformViewManager::PlatformViewManager(BinaryMessenger* binary_messenger)
+    : channel_(std::make_unique<MethodChannel<EncodableValue>>(
+          binary_messenger,
+          kChannelName,
+          &StandardMethodCodec::GetInstance())) {
+  channel_->SetMethodCallHandler(
+      [this](const MethodCall<EncodableValue>& call,
+             std::unique_ptr<MethodResult<EncodableValue>> result) {
+        const auto& args = std::get<EncodableMap>(*call.arguments());
+        if (call.method_name() == "create") {
+          const auto& type =
+              std::get<std::string>(args.find(EncodableValue("type"))->second);
+          const auto& id =
+              std::get<std::int32_t>(args.find(EncodableValue("id"))->second);
```

Comment:
Are these required parameters? If yes, could we make this return an error if the parameters aren't provided? This would prevent a crash if we get the args wrong

Reply:
Yes, they are required. That may be a good idea.